news/_posts/core-weekly/2019-01-14-coreweekly-week-02-2019.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 02 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-01-14 16:00:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 07th to Sunday 13th of January 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week, two new repositories have been added to the PrestaShop projet on GitHub:
+
+- [PhpPsInfo](https://github.com/PrestaShop/php-ps-info/), a tool to check if a server has all Apache and PHP prerequisites before starting to install a PrestaShop online store on it
+- [Circuit Breaker](https://github.com/PrestaShop/circuit-breaker/), a project to make PrestaShop online stores able to work well even when third party services are not available (payment solutions, solutions using the webservices/API, Addons, ...)
```

Comment:
```suggestion
- [Circuit Breaker](https://github.com/PrestaShop/circuit-breaker/), a library that allows applications to better handle cases when third party services (payment solutions, solutions using the webservices/API, Addons, ...) go offline. It will be progressively be introduced in PrestaShop and its modules.
```